tfe/data_source_variable_set_test.go

```diff
@@ -112,5 +120,5 @@ data "tfe_variable_set" "foobar" {
   name = tfe_variable_set.foobar.name
 	organization = tfe_variable_set.foobar.organization
 	depends_on = [tfe_variable.envfoo]
-}`, rInt, rInt, rInt)
+}`, rInt, rInt, rInt, rInt)
```

Comment:
🤷‍♀️  maybe it's this because project_ids aren't an argument to the data source bit?

```suggestion
}`, rInt, rInt, rInt
```